install_config/install/docker_registry.adoc

```diff
@@ -20,26 +20,47 @@ link:../../architecture/infrastructure_components/image_registry.html#integrated
 Docker registry] that can be deployed in your OpenShift environment to locally
 manage images.
 
-To make full use of the registry, you must create a
-link:../../architecture/core_concepts/projects_and_users.html#users[regular user]
-with the *system:registry* role, if one does not already exist, using
-your preferred
-link:../../admin_guide/configuring_authentication.html[identity provider].
-This is because
-link:../../architecture/core_concepts/projects_and_users.html#users[system users],
-such as *system:admin*, cannot obtain access tokens, and therefore
+[[prerequisites-for-access]]
+=== Prerequisites for Access
```

Comment:
This can be a [level 1 section](http://asciidoctor.org/docs/asciidoc-writers-guide/#section-titles) (i.e., use `==`), unless you had a reason for it to be a subsection of the Overview?